packages/client/lib/index.ts

```diff
@@ -284,7 +299,13 @@ export class RelayClient extends EventEmitter {
         this.#decodeMessage.bind(this),
       );
     }
-
+    if (
+      this.connection.readyState === WebSocket.OPEN &&
```

Comment:
we should use the "open" event and then authenticate, instead of checking the state each time. It think. So we should remove this.